### PR TITLE
fix(quantic): failing create quantic package workflow

### PR DESCRIPTION
**Pull request by SimonMilord**
[SFINT-5191](https://coveord.atlassian.net/browse/SFINT-5191)

**In this PR:**

- Fixed the failing Create Quantic Package workflow => it was failing because the repository was not checked out when checking for node version in the `.nvmrc` file.
- Added report-status action in the workflow to print the build status in our Slack channel after every run => this should allow us to monitor the build status more closely.

image

[SFINT-5191]: https://coveord.atlassian.net/browse/SFINT-5191?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/create-quantic-package.yml [hunk 1]
```diff
@@ -52,3 +52,19 @@ jobs:
...
+    timeout-minutes: 30
+    steps:
+      - run: |
+          url="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+          if [[ ${{ contains(needs.*.result, 'failure') }} == "false" ]]; then
+            message="🟢 Create Quantic Package build run success"
+          else
+            message="🔴 Create Quantic Package build run failure"
+          fi
+          curl -X POST -H "Content-Type: application/json" -d "{\"message\": \"$message\", \"url\": \"$url\"}" ${{ secrets.SLACK_WEBHOOK_CREATE_QUANTIC_PACKAGE }}
```
Comment (louis-bompart) [inline .github/workflows/create-quantic-package.yml:70]: - [ ] @louis-bompart add the secret.

File: .github/workflows/create-quantic-package.yml [hunk 2]
```diff
@@ -15,11 +15,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     steps:
+      - name: Check Out Repository
```
Comment (erocheleau) [inline .github/workflows/create-quantic-package.yml:18]: lol d'oh good catch

File: .github/workflows/create-quantic-package.yml (hunk 1 above)
Comment (erocheleau) [reply .github/workflows/create-quantic-package.yml:70]: Wait I recognize this code ;)

**Review by erocheleau — APPROVED** [review]
Very nice catch on why the file wasn't there

**Review by btaillon — APPROVED** [review]

**Pull request merged**